src/main/java/org/archive/hadoop/jobs/WEATGenerator.java

```diff
@@ -93,17 +97,25 @@ public void map( Text key, Text value, OutputCollector output, Reporter reporter
         String inputBasename = inputPath.getName();
         String watOutputBasename = "";
         String wetOutputBasename = "";
+        String cdxWatOutputBasename = "";
+        String cdxWetOutputBasename = "";
 
         if(path.endsWith(".gz")) {
           watOutputBasename = inputBasename.substring(0,inputBasename.length()-3) + ".wat.gz";
           wetOutputBasename = inputBasename.substring(0,inputBasename.length()-3) + ".wet.gz";
+          cdxWatOutputBasename = inputBasename.substring(0,inputBasename.length()-3) + ".cdxwat.gz";
+          cdxWetOutputBasename = inputBasename.substring(0,inputBasename.length()-3) + ".cdxwet.gz";
         } else {
           watOutputBasename = inputBasename + ".wat.gz";
           wetOutputBasename = inputBasename + ".wet.gz";
+          cdxWatOutputBasename = inputBasename + ".cdxwat.gz";
+          cdxWetOutputBasename = inputBasename + ".cdxwet.gz";
         }
 
         String watOutputFileString = basePath.toString() + "/wat/" + watOutputBasename;
         String wetOutputFileString = basePath.toString() + "/wet/" + wetOutputBasename;
+        String cdxWetOutputFileString = basePath.toString() + "/cdxwet/" + cdxWetOutputBasename;
```

Comment:
This is a fixed output path. Do we want to have the CDX files for WAT/WET there?

The configuration for the CDX indexing uses different output paths, cf. https://github.com/commoncrawl/webarchive-indexing/blob/main/run_index_hadoop.sh